schema/encoding/jsonschema/jsonschema.go

```diff
@@ -41,7 +44,25 @@ func (ew errWriter) write(b []byte) {
 	_, ew.err = ew.w.Write(b)
 }
 
-// ValidatorToJSONSchema takes a validator and renders to JSON
+// boundariesToJSONSchema writes JSON Schema keys and values based on b, prefixed by a comma and without curly braces,
+// to w. The prefixing comma is only written if at least one key/value pair is also written.
+func boundariesToJSONSchema(w io.Writer, b *schema.Boundaries) error {
+	if b == nil {
+		return nil
+	}
+	ew := errWriter{w: w}
+
+	if !math.IsNaN(b.Min) && !math.IsInf(b.Min, -1) {
+		ew.writeFormat(`, "minimum": %s`, strconv.FormatFloat(b.Min, 'E', -1, 64))
+	}
+	if !math.IsNaN(b.Max) && !math.IsInf(b.Max, 1) {
+		ew.writeFormat(`, "maximum": %s`, strconv.FormatFloat(b.Max, 'E', -1, 64))
+	}
+	return ew.err
+}
```

Comment:
> I mean, nan is triggered by strings

What do you mean it's triggered by strings? Where, and how? E.g `strconv.ParseFloat()` would return an error and a zero value, not NaN: https://play.golang.org/p/Xx50p-LgL7

> so is it a good idea to silently accept it?

I don't think there are too many ways people can set NaN by mistake. They may get an `NaN` if they do `math.Acos(1.1)` etc., but that doesn't really sound like a common pattern for setting float limits? I would assume the most reasonable way people could set a NaN value, would be if they did so deliberately via `math.NaN()`...

> Yes but why accept it?
Currently I made the JSON Schema accept it simply because the `schema` package already accepts it (silently), and we should handle the value somehow....